install-paperless-ngx.sh

```diff
@@ -71,7 +71,16 @@ if ! docker stats --no-stream &> /dev/null ; then
 	sleep 3
 fi
 
-default_time_zone=$(timedatectl show -p Timezone --value)
+# Added handling for timezone for busybox based linux, not having timedatectl available (i.e. QNAP QTS)
+# if neither timedatectl or /etc/TZ is suceeding, defaulting to GMT.
+if  command -v timedatectl &> /dev/null ; then
+        default_time_zone=$(timedatectl show -p Timezone --value)
+elif [ -f /etc/TZ ] && [ -f /etc/tzlist ] ; then
+        TZ=`cat /etc/TZ`
+        default_time_zone=$(grep -B 1 -m 1 $TZ /etc/tzlist | head -1 | cut -f 2 -d =)
+else
+        default_time_zone="Europe/London"
```

Comment:
- This should not use Europe/London, but [Etc/UTC](https://en.wikipedia.org/wiki/List_of_tz_database_time_zones#UTC)
- The user should be informed this has happened

Reply:
Any preferences on how this information should look like?

i.e. echo "Unable to detect timezone, defaulting to Etc/UTC"?

Reply:
That's fine

Reply:
solved with https://github.com/paperless-ngx/paperless-ngx/pull/6336/commits/41a5010bfef127f4976d0ebca6ffc1d34f695e6b